end_to_end_testing/setup_helm2.sh

```diff
@@ -0,0 +1,12 @@
+source "$( cd -P "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/setup_common.sh
+
+echo "Installing Helm"
+curl -sL https://storage.googleapis.com/kubernetes-helm/helm-v2.14.2-linux-amd64.tar.gz | tar xzv linux-amd64/helm
```

Comment:
This version is also out of date. Probably need a better way to keep track of these for the future. Or run multiple version tests by parameterizing this  and exposing the parameter to Circle config.

Reply:
Saw that. Elected not to tackle it at this time.